dev_apps/music-nga/js/app.js

```diff
@@ -112,6 +163,10 @@ function setHeaderTitle(title) {
   window.requestAnimationFrame(() => headerTitle.textContent = title);
 }
 
+function setBackButtonHidden(hidden) {
+  header.els.actionButton.style.visibility = hidden ? 'hidden' : 'visible';
+}
```

Comment:
Ahh.. I know why I did this. The button is in the <gaia-header> shadow DOM and I can't apply an external CSS class to it.